Sources/Authentication/AppAuthSession.swift

```diff
@@ -123,14 +123,13 @@ public final class AppAuthSession: LoginSession {
     
     private func generateTokenResponse(token: OIDTokenResponse, authState: OIDAuthState) throws -> TokenResponse {
         guard let accessToken = token.accessToken,
-              let idToken = token.idToken,
               let tokenType = token.tokenType,
               let expiryDate = token.accessTokenExpirationDate else {
             throw LoginError.generic(description: "Missing authState property")
         }
         return TokenResponse(accessToken: accessToken,
                              refreshToken: authState.refreshToken,
-                             idToken: idToken,
+                             idToken: token.idToken,
```

Comment:
Out of curiosity, why does `refreshToken` come from the `authState` but `idToken` comes from the `token`? (As discussed on Slack, we don't currently return a `refresh_token` in the token response anyway, so it shouldn't matter)

Reply:
The AppAuth-iOS package [defines the refresh token as part of the OIDAuthState object](https://github.com/openid/AppAuth-iOS/blob/master/Sources/AppAuthCore/OIDAuthState.h#L62-L68):
```obj-c
/*! @brief The most recent refresh token received from the server.
    @discussion Rather than using this property directly, you should call
        @c OIDAuthState.performActionWithFreshTokens:.
    @remarks refresh_token
    @see https://tools.ietf.org/html/rfc6749#section-5.1
 */
@property(nonatomic, readonly, nullable) NSString *refreshToken;
```
Probably because this type is persisted to use `OIDAuthState.performActionWithFreshTokens` as mentioned in the API description above.
When we get on to using the refresh token I imagine the way we retrieve and exchange it for access tokens will be subject to change.